mm2src/crypto/src/mnemonic.rs

```diff
@@ -144,10 +143,23 @@ mod tests {
         let mnemonic = "tank abandon bind salon remove wisdom net size aspect direct source fossil";
         let password = "password";
 
-        // Verify that the mnemonic is valid
-        let parsed_mnemonic = Mnemonic::parse_normalized(mnemonic);
-        assert!(parsed_mnemonic.is_ok());
-        let parsed_mnemonic = parsed_mnemonic.unwrap();
+        // Encrypt the mnemonic
+        let encrypted_data = encrypt_mnemonic(mnemonic, password);
+        assert!(encrypted_data.is_ok());
+        let encrypted_data = encrypted_data.unwrap();
+
+        // Decrypt the mnemonic
+        let decrypted_mnemonic = decrypt_mnemonic(&encrypted_data, password);
+        assert!(decrypted_mnemonic.is_ok());
+        let decrypted_mnemonic = decrypted_mnemonic.unwrap();
+
+        // Verify if decrypted mnemonic matches the original
+        assert_eq!(decrypted_mnemonic, mnemonic);
+    });
+
+    cross_test!(test_encrypt_decrypt_words, {
```

Comment:
I guess a better test name would be `test_encrypt_decrypt_non_bip39_mnemonic`

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2312/commits/d52cf50b9fe7ee3b64d89b081c0259c9e44466c7